docs/source/kedro.logging.RichHandler.rst

```diff
@@ -0,0 +1,9 @@
+kedro.logging.RichHandler
+=========================
+
+.. currentmodule:: kedro.logging
+
+.. autoclass:: RichHandler
+   :members:
+   :undoc-members:
+   :inherited-members:
```

Comment:
Maybe we'll have to remove the `:undoc-members:` and `:inherited-members:`

Reply:
Thank you! This seems to work. Sphinx is a puzzle, I cannot reproduce the same error in my local environment, but I am glad it's fixed now.

Reply:
I can't believe it worked 😂 I guess I'm a Sphinx black belt now 🥷